src/Polling/Infrequent/TestService.cs

```diff
@@ -18,6 +20,9 @@ public async Task<string> GetServiceResultAsync(CancellationToken cancellationTo
             return "OK";
         }
 
-        throw new TestServiceException("Service is down");
+        // Set the error as BENIGN to indicate it is an expected error.
+        // BENIGN errors have activity failure logs downgraded to DEBUG level and
+        // do not emit activity failure metrics.
```

Comment:
The test service is meant to be non-Temporal related. You wouldn't have control over the exception thrown by your upstream service. Rather, I think we should leave everything as is and just change `InfrequentPollingActivities` to throw a new benign exception wrapping this one.

Reply:
done